### PR TITLE
Add styles and components for event category screen

### PR DESCRIPTION
**Pull request by yongmin86k**
# Add styles and components for event category screen

- Create and style the components for the screen
    - Add category lists
    - Add sort screen when it's pressed
- Install `moments.js` to render the date

image
image

### CODE REVIEW

File: client/js/components/CardEvent/CardEvent.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+              <View style={styles.detailsWrapper}>
+                <View style={styles.detailsInfo}>
+                  <Text style={styles.caption}>Date</Text>
+                  <Text style={styles.detailContent}>
+                    {`${moment(date).format('MMM D, Y')} ${
+                      parseInt(startAt) > 12 ? 12 - 12 : parseInt(startAt)
+                    }-${
+                      parseInt(endAt) > 12
+                        ? parseInt(endAt) - 12 + 'PM'
+                        : parseInt(endAt) + 'AM'
```
Comment (siddharthparmar7) [inline client/js/components/CardEvent/CardEvent.js]: Please define what is 12?

File: client/js/screens/EventCategory/EventCategory.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+  const animateHeader = () => {
+    Animated.timing(headerAnimation, {
+      duration: 500,
+      toValue: isAnimated ? {x: 0, y: -88} : {x: 1, y: 0},
+    }).start();
+  };
+  //   console.log(eventInfo);
+  return (
+    <>
+      {!paddingHeight && <View style={{height: 88}} />}
```
Comment (siddharthparmar7) [inline client/js/screens/EventCategory/EventCategory.js]: Please define what is 88, -88. Also, remove commented code.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Nice work. Please make the changes.

File: client/js/components/CardEvent/CardEvent.js (hunk 1 above)
Comment (yongmin86k) [reply client/js/components/CardEvent/CardEvent.js]: Updated~

File: client/js/screens/EventCategory/EventCategory.js (hunk 2 above)
Comment (yongmin86k) [reply client/js/screens/EventCategory/EventCategory.js]: Defined and removed

**Comment by siddharthparmar7** [comment]
Please resolve merge conflicts.

**Comment by siddharthparmar7** [comment]
Please solve merge conflicts.

**Pull request merged**